test_ehr_analysis.py

```diff
@@ -0,0 +1,62 @@
+import ehr_analysis
+import pytest
+
+
+def test_parse_data_patient():
+    """
+    A test function that checks whether the parse_data function returns the correct list for patient data
+    """
+    assert [
+        "64182B95-EB72-4E2B-BE77-8050B71498CE",
+        "Male",
+        "1952-01-18 19:51:12.917",
+        "African American",
+        "Separated",
+        "English",
+        "13.03",
+    ] in ehr_analysis.filename
```

Comment:
`ehr_analysis.filename` does not exist. One should be able to run `pytest test_ehr_analysis.py` after cloning the repo.

Reply:
The tests still do not run. There is no `ehr_analysis.patient`, among other issues..